### ISSUE TITLE
Generar capa de elevación para el modelo de vulnetabilidad #PAPIIT

### ISSUE THREAD

**Issue by vichdzgeo** [lancis-apc/espejos-lancis#41]

**Comment by vichdzgeo** [comment]
@CarlosCruzVz proporcionará los valores mínimo y máximo de elevación. 

- Recortar el raster de elevación únicamente a la zona de estudio de los 3 kms!
- Generar la estadística de los valores y obtener el mínimo y el máximo!

**Comment by CarlosCruzVz** [comment]
Los valores de las estadistica maxima es 25 y la estadistica minima es -8
para la capa de elevación recortada a 3 km

@vichdzgeo @paola-gp @yosunemiquela @serranoycandela

**Comment by paola-gp** [comment]
@CarlosCruzVz: cual es el tamaño de pixel de la capa?

**Comment by vichdzgeo** [comment]
@paola-gp 

las funciones de valor se localizan en: 
PAPIIT/pap_modelos/vul_costera/fv

**Comment by CarlosCruzVz** [comment]
La capa se ubica en la siguiente ruta
Dropbox/sig_papiit/procesamiento/fisicas/elevacion/elevacion_3km.tif
la resolucion es de 30m el pixel

**Comment by paola-gp** [comment]
@CarlosCruzVz: cuál es el archivo de la función de valor?

**Comment by CarlosCruzVz** [comment]
Esta en 
PAPIIT/pap_modelos/vul_costera/fv/

r_fis_elev.json
r_fis_elev.png
s_fis_elev.json
s_fis_elev.png

El de exposicion no lo subieron por que dijeron que como ya los habiamos visto no tenia caso
pero si no mal recuerdo es el logistica con centro=5 y k=0.083

**Comment by paola-gp** [comment]
@CarlosCruzVz: la función de elevación con base en el Json era concava decreciente para los dos casos. Ya reemplacé los archivos anteriores con las versiones actualizadas. r_fis_elev.json
r_fis_elev.png
s_fis_elev.json
s_fis_elev.png

Nota: son la misma ecuación, se puede hacer una vez y asegurarse de que queda con los nombres correctos.

**Comment by CarlosCruzVz** [comment]
@paola-gp: ok, entonces las funciones son identicas no hay diferencias?
según yo
exposicion era logistica
sensibilidad era concava creciente
resiliencia era concava decreciente

Otra duda la funcion de valor de exposicion como seria su actualización

**Comment by paola-gp** [comment]
@CarlosCruzVz: en la carpeta no encontré el Json o png de elevación para exposición. ¿Ustedes lo tienen en otro lado? estoy doble checando sensibilidad y resiliencia

**Comment by CarlosCruzVz** [comment]
@paola-gp : no, el de exposición se hizo en el taller anterior y se lo mandaron a fidel para que él nos diera la formula de la función

@yosunemiquela

**Comment by paola-gp** [comment]
@CarlosCruzVz: ya se verificaron y quedaron como sigue:

- sensibilidad es concava creciente (se corrigió)
- resiliencia es concava decreciente 
- quedo en espera a que rastreen e_fis_elev.json y lo pongan /pap_modelos/vul_costera/fv/
-

**Comment by CarlosCruzVz** [comment]
@paola-gp : perfecto, quedamos al pendiente

**Comment by paola-gp** [comment]
@CarlosCruzVz: si ustedes no saben dónde está, cómo sabes que fue una función logística, está en algún issue?

**Comment by paola-gp** [comment]
@CarlosCruzVz y @yosunemiquela: Creemos que es esta la función original de e_fis_elev, pero necesitamos confirmación para estar seguros de que no hay error

{"layer": "elevacion", "center": "5", "min": "0", "max": "10", "k": "0.08349999999999999", "show_map": "False", "function_name": "logistic"}

**Comment by paola-gp** [comment]
@CarlosCruzVz y @yosunemiquela: si podemos confirmar que el json anterior corresponde a la ecuación que hicieron originalmente, entonces en la carpeta ya puse la transformación con los nuevos máximos y mínimos en json y png

**Comment by yosunemiquela** [comment]
Perfecto.Gracias @paola-gp

**Comment by vichdzgeo** [comment]
@CarlosCruzVz

Dada la capa reportada con los metadatos entregados, he encontrado una diferencia significante en el insumo de la capa de elevación.

El valor mínimo y máximo según la estadística de la capa 

Ruta = /sig_papiit/entregables/exposicion//fisica/elev_yuc/i_elev_yuc.tif"

son : 
valor mínimo 
-40.00000

valor máximo 
44.00000

causando que en las funciones de valor de resiliencia y sensibilidad sus valores no estén entre 0 y 1.
y aunque los valores  la función de valor de elevación del criterio de exposición si se encuentran entre 0 y 1, no es del todo correcto.

la cuestión aquí parte desde la elección del insumo, el cual según lo declarado en el linaje de la capa 
se ocupa este recurso - >https://explorer.earthengine.google.com/#detail/USGS%2FSRTMGL1_003

¿Por que se elige ese insumo?


 @paola-gp @yosunemiquela

**Comment by CarlosCruzVz** [comment]
![Captura](https://user-images.githubusercontent.com/43582616/58049395-5a87fc80-7b12-11e9-92a5-4ddd5dcf7ef5.JPG)
La captura de los valores menciona como valor minimo -8 y max 25
el insumo se utilizó por practicidad, pues se tenia una capa de pendientes y este insumo es de elevación, ademas se recorto a la extensión de todas las demas
@vichdzgeo

**Comment by vichdzgeo** [comment]
@CarlosCruzVz si, en efecto puede uno confundirse, ya que qgis lo que hace es crear una rápida estadística de los valores mínimos y máximos, para visualizar los datos completos, justo en la ventana que muestras hay una opción del lado derecho (a mitad de la ventana) que se llama precisión. la opción por default es la que muestras **Estimar (más rápido) ** pero para conocer los datos completos se tiene que cambiar la opción estimar por **actual (más lento)**

![image](https://user-images.githubusercontent.com/38017509/58108759-07658680-7bb2-11e9-82ad-f503e4a89c33.png)

**Comment by vichdzgeo** [comment]
@CarlosCruzVz  por otra parte, ayer me reuní con @yosunemiquela y lo mejor será entonces utilizar el Modelo digital de elevaciones de INEGI.  a 30 metros

**Comment by CarlosCruzVz** [comment]
@vichdzgeo 
La capa de INEGI es el Continuo de Elevaciones?
Por otro lado, quedaré al pendiente de la funcion de valor que originan los valores maximos y minimos reales, para rehacer la capa

**Comment by vichdzgeo** [comment]
@CarlosCruzVz Asi es, te parece si la descargas, la alineas con el extent que tenemos  y la recortas a la zona de estudio. y ya en conjunto verificamos el mínimo y el máximo de valores para re calcular las funciones de valor

**Comment by CarlosCruzVz** [comment]
![elevacion](https://user-images.githubusercontent.com/43582616/58115830-8bbf0600-7bc0-11e9-8146-05faa4964100.png)
ya esta cortada y en la imagen muestra los datos min y max reales
Ruta: Dropbox/sig_papiit/procesamiento/exposicion/fisica/elev_yuc/cem_yuc_30m_clip.tif

**Comment by vichdzgeo** [comment]
@CarlosCruzVz  no encuentro nada en la ruta indicada, tal vez se este sincronizando, podrías verificar de favor.

**Comment by CarlosCruzVz** [comment]
@vichdzgeo 
Listo ya esta en la carpeta

**Comment by vichdzgeo** [comment]
@CarlosCruzVz  ya verifique la capa, pero no estaba alineada, en la misma carpeta puse el código y genere una capa con el extent1 para que este alineado, del insumo original.

podrías recortar de nuevo esta capa creada **cem_yuc_30m_v2.tif** y calcular de nuevo el máximo y mínimo del recorte

**Comment by vichdzgeo** [comment]
## IMPORTANTE
este issue se divide en tres issue's. una por capa.

elevación_exposición #57 
elevación_resiliencia #58 
elevación_sensibilidad #59
